selfdrive/car/volkswagen/values.py

```diff
@@ -294,6 +294,12 @@ class CAR(Platforms):
     chassis_codes={"A3"},
     wmis={WMI.VOLKSWAGEN_USA_CAR},
   )
+  VOLKSWAGEN_JETTA_SPORTWAGEN = VolkswagenPQPlatformConfig(
```

Comment:
Update this and all other CAR identifier references

```suggestion
  VOLKSWAGEN_JETTA_MK6 = VolkswagenPQPlatformConfig(
```